lib/cisco_node_utils/cmd_ref/bridge_domain.yaml

```diff
@@ -0,0 +1,44 @@
+# bridge_domain
+# bridge_domain feature is available only on n7k
+---
+_exclude: [N3k, N5k, N6k, N9k]
+
+all_bds:
+  multiple: true
+  get_command: "show bridge-domain"
```

Comment:
Is it possible to get the information you need from running-config alone? We have run into numerous problems when using non-show-run methods to get cli state, especially when trying to support new platforms. I know that bridge-domain is an unusually messy cli but please explore the running-config approach.